drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt_scan.c

```diff
@@ -65,13 +65,14 @@ int nrf_wifi_disp_scan_zep(const struct device *dev, struct wifi_scan_params *pa
 	}
 
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
-	fmac_dev_ctx = rpu_ctx_zep->rpu_ctx;
-
 	if (!rpu_ctx_zep) {
 		LOG_ERR("%s: rpu_ctx_zep is NULL\n", __func__);
 		goto out;
 	}
 
+	fmac_dev_ctx = rpu_ctx_zep->rpu_ctx;
+
```

Comment:
nit: Double newline.

Reply:
oops, just saw this comment, will fix in another PR, thanks.